module/VuFind/src/VuFind/Auth/Manager.php

```diff
@@ -704,11 +755,43 @@ public function login($request)
         // Update user object
         $this->updateUser($user);
 
+        if ($request->getPost()->get('remember_me') && $this->supportsPersistentLogin()) {
+            try {
+                $this->loginToken->createToken($user, '', $this->sessionManager->getId());
+            } catch (\Exception $e) {
+                throw new AuthException('authentication_error_technical', 0, $e);
```

Comment:
It would be helpful to add `$this->logError((string)$e);` here so there is easier visibility of the underlying error when logging is configured.